source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -50,7 +50,7 @@ export abstract class Dialog {
             this._overlayElement.className = "acd-dialog-overlay";
             this._overlayElement.onclick = (e) => {
                 // clicks on the overlay window should dismiss the dialog
-                this.close();
+                this.preventDefaultAction && this.close();
```

Comment:
Can we please write it this way instead
```
if (this.preventDefaultAction) {
  this.close();
}
```

Also, `preventDefaultAction` isn't very descriptive. Prevent what action? Maybe rename into `preventLightDismissal`.

Reply:
Updated the variable name. Addressed the review comments